jobs/director/spec

```diff
@@ -144,6 +144,9 @@ properties:
   director.enable_nats_delivered_templates:
     description: When true, rendered templates will be sent over NATs
     default: false
+  director.enable_short_lived_nats_bootstrap_credentials:
+    description: When true, NATS bootstrap credentials will be short lived on new VMs
+    default: false
```

Comment:
@beyhan & @rkoster, what do you think about flipping this so it's enabled by default?

We've seen very little cost to turning this on, and secure by default seems like the correct security posture.

Reply:
How much time does it add to a typical vm_creation (absolute value and percentage of total)?

Reply:
If it adds a few seconds it makes sense to enable it by default, If it's more than say half a minute I'm a bit hesitant.

Reply:
On the other hand, they can always disable it, we just need to make sure to properly communicate about it in the release notes.

Reply:
Having this enabled by default will help us to get faster feedback for the feature and I think our strategy is to provide a secure configuration by default. In case it has significant performance implications we we should at least document this so that the operators can decide.

Reply:
@rkoster a deployment with the feature deactivated for a couple of vms take 1:01 minutes to deploy. If the features is activated then it takes 1:52 minutes.
The VM creation for a single vm goes from:
Task 335 | 21:55:28 | Creating missing vms: jammy-group/e3a8c572-fc51-4055-abce-09828ceecf03 (0) (**00:00:45**)
Task 333 | 21:51:26 | Creating missing vms: jammy-group/efd9ee27-82d6-424a-87c3-a52da210385d (0) (**00:01:35**)
However,  this gets minimized if the deployment creates the VMs in parallel, deploying around 50 VMs takes nearly 03:49 minutes to deploy when the feature is deactivated, and around 3:55 minutes when the feature is activated. 
So the parallelism reduces the total time to deploy many VMs. (At least in GCP, which uses light stemcells.)

Reply:
From 03:49 to 03:55 on the 50 VM deploy seems acceptable to me. So yeah let's enable it by default.

Reply:
Cool, the feature is now enabled by default. All the force pushes were made to squash all the commits to tune the code, so please, don't mind them.